ansible/roles/docker/templates/etc/default/docker.default

```diff
@@ -4,8 +4,11 @@
 #DOCKER="/usr/local/bin/docker"
 
 # Use DOCKER_OPTS to modify the daemon startup options.
-#DOCKER_OPTS="--dns 8.8.8.8 --dns 8.8.4.4"
-DOCKER_OPTS="--storage-driver=devicemapper"
```

Comment:
Yea

```
Storage Driver: aufs
 Root Dir: /var/lib/docker/aufs
 Backing Filesystem: extfs
 Dirs: 26
 Dirperm1 Supported: false
```

I'm not familiar with all the different storage drivers, but after reading through the descriptions, it seems like `aufs` is the best fit for empire